### PR TITLE
ijq: 1.1.0 -> 1.1.1

### PR DESCRIPTION
**Pull request by mattpolzin**
## Description of changes

This PR includes a patch version bump, nix formatting, relocation to pkgs/by-name, and finally I'd like to volunteer to help maintain this package so I've added myself to the list of maintainers.

Version update changes: https://github.com/gpanders/ijq/compare/v1.1.0...v1.1.1.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by mattpolzin** [comment]
Result of `nixpkgs-review pr 333783` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ijq</li>
  </ul>
</details>

**Comment by mattpolzin** [comment]
Result of `nixpkgs-review pr 333783` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ijq</li>
  </ul>
</details>

File: pkgs/by-name/ij/ijq/package.nix [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
   pname = "ijq";
-  version = "1.1.0";
+  version = "1.1.1";
 
   src = fetchFromSourcehut {
     owner = "~gpanders";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-KtWJwIofMKW+03DFY4UWf3ni1DKuH289svh8iOPo1so=";
+    sha256 = "sha256-rnSpXMadZW6I+7tIYqr1Cb4z00gdREsqin/r6OXaDMA=";
```
Comment (drupol) [inline pkgs/by-name/ij/ijq/package.nix:19]:
```suggestion
    hash = "sha256-rnSpXMadZW6I+7tIYqr1Cb4z00gdREsqin/r6OXaDMA=";
```

File: pkgs/by-name/ij/ijq/package.nix [hunk 2]
```diff
@@ -1,21 +1,37 @@
-{ buildGoModule, fetchFromSourcehut, lib, jq, installShellFiles, makeWrapper, scdoc }:
+{
+  buildGoModule,
+  fetchFromSourcehut,
+  lib,
+  jq,
+  installShellFiles,
+  makeWrapper,
```
Comment (drupol) [inline pkgs/by-name/ij/ijq/package.nix:7]:
```suggestion
  makeBinaryWrapper,
```

For 2 reasons: a small performance gain and a better compatibility on macOS.

**Review by drupol — CHANGES_REQUESTED** [review]

File: pkgs/by-name/ij/ijq/package.nix [hunk 3]
```diff
@@ -1,21 +1,37 @@
...
+}:
 
 buildGoModule rec {
   pname = "ijq";
-  version = "1.1.0";
+  version = "1.1.1";
 
   src = fetchFromSourcehut {
     owner = "~gpanders";
     repo = pname;
```
Comment (drupol) [inline pkgs/by-name/ij/ijq/package.nix:17]: Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

**Comment by mattpolzin** [comment]
I addressed PR feedback in the same commit as I ran the formatter.

**Comment by mattpolzin** [comment]
Result of `nixpkgs-review` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ijq</li>
  </ul>
</details>

**Review by drupol — APPROVED** [review]

**Comment by mattpolzin** [comment]
Result of `nixpkgs-review pr 333783` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ijq</li>
  </ul>
</details>

**Pull request merged**